src/containers/NavBar.tsx

```diff
@@ -22,7 +24,46 @@ const Root = styled.div<{ theme: Theme; position: "relative" | "fixed" }>`
   & > button {
     padding: 0 1em;
   }
+  .nav-button {
+    display: none !important;
+    position: absolute;
+    right: 2em;
+    top: 2em;
+    transform: translate(0, -50%);
+  }
+
+  @media (max-width: 768px) {
+    .nav-button {
+      display: block !important;
+    }
+    :not(.nav-open) {
```

Comment:
Im having the same issue after recompiling. It appears this not selector isnt working. Maybe its a compatibility issue https://developer.mozilla.org/en-US/docs/Web/CSS/:not#Browser_compatibility